src/org/labkey/test/tests/ExpTest.java

```diff
@@ -163,17 +166,17 @@ public void testSteps()
         waitForElement(Locator.xpath("//span[contains(text(), 'Reset to Default')]"), defaultWaitForPage);
         int lastFieldIndex = getElementCount(Locator.xpath("//input[starts-with(@name, 'ff_label')]")) - 1;
         Locator lastField = Locator.xpath("//input[@name='ff_label" + lastFieldIndex + "']");
-        click(lastField);
-        click(Locator.xpath("//span").append(Locator.lkButton("Alias Field")));
-        selectOptionByText(Locator.name("sourceColumn"), "RowId");
-        click(Locator.xpath("//span").append(Locator.lkButton("OK")));
+        domainRow = designerPage.fieldsPanel().getField(lastFieldIndex);
```

Comment:
is this lastFieldIndex valid for the new UI designer? Looks like it is based off of looking for the count of "ff_label" as an input name. I don't think we use those type of names anymore. Should that line above changes as well? Or should we instead be using the domainDesigner test helper to get the count of the fields (I believe it has that helper on the DomainFormPanel)?